cl/settings/third_party/sentry.py

```diff
@@ -13,6 +15,31 @@
 ignore_logger("internetarchive.session")
 ignore_logger("internetarchive.item")
 
+
+def fingerprint_event(
```

Comment:
I think the name could be slightly more descriptive...

maybe something like ... `fingerprint_sentry_error`